frontend/openapi/apis/OAuthApi.ts

```diff
@@ -0,0 +1,74 @@
+// @ts-nocheck
```

Comment:
This file should not be generated - both endpoints are redirects that meant to be executed by browser